packages/wonder-blocks-icon-button/components/icon-button-core.js

```diff
@@ -0,0 +1,143 @@
+// @flow
+import React from "react";
+import {StyleSheet} from "aphrodite";
+
+import Color, {mix, fade} from "wonder-blocks-color";
+import {addStyle} from "wonder-blocks-core";
+import type {ClickableHandlers} from "wonder-blocks-core";
+import type {SharedProps} from "./icon-button.js";
+
+type Props = SharedProps &
+    ClickableHandlers & {
+        hovered: boolean,
+        focused: boolean,
+        pressed: boolean,
```

Comment:
These should be `optional` since flow will complain about them not being used in the generated snapshot tests.  See https://github.com/Khan/wonder-blocks/pull/88.

Reply:
Sorry I'm confused. They are used in `docs.md`?

Reply:
My bad, disregard.  I ran into this issue with the existing buttons/links and I assumed that button links would have the same issue.